policyengine_us/variables/gov/local/ca/la/general_relief/la_general_relief.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class la_general_relief(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Los Angeles County General Relief"
+    definition_period = MONTH
+    defined_for = "la_general_relief_eligible"
+    reference = "https://drive.google.com/file/d/1Oc7UuRFxJj-eDwTeox92PtmRVGnG9RjW/view?usp=sharing"
+
+    def formula(spm_unit, period, parameters):
+        married = add(spm_unit, period, ["is_married"])
+        p = parameters(period).gov.local.ca.la.general_relief
+        gr_amount = where(married, p.amount.married, p.amount.single)
+        # If filers are receiving the housing subsidy
+        # they are obligate to commit an amount of their GR towards rent
+        # which is deducted here and added to the housing subsidy amount
+        receive_housing_subsidy = (
+            spm_unit("la_general_relief_housing_subsidy", period) > 0
+        )
+        rent_contributions = where(
+            married,
+            p.housing_subsidy.rent_contribution.married,
+            p.housing_subsidy.rent_contribution.single,
+        )
+
```

Comment:
make a separate variable, plus la_general_relief_base, then have la_general_relief use adds/subtracts. base needs to phase out for recipients, linearly by dividing the excess over the phase-out start over the phase-out width
```suggestion
        return rent_contributions * receives_housing_subsidy
```